docusaurus/docs-internals/editor-controls/multivalue-controls/iconsPicker.md

```diff
@@ -203,28 +283,52 @@ const getToolbarContols = ({ getValue }) => {
       type: "select",
       choices: [
         { title: "Youtube", value: "youtube" },
-        { title: "Custom", value: "custom" },
-      ],
+        { title: "Custom", value: "custom" }
+      ]
     },
     {
-      id: "icns",
+      id: "icons",
       type: "iconsPicker",
       disabled: videoType === "custom",
       choices: [
         {
           title: "Item 1",
           value: "val1",
-          icon: "nc-star",
+          icon: "nc-star"
         },
         {
           title: "Item 2",
           value: "val2",
-          icon: "nc-line",
-        },
-      ],
+          icon: "nc-line"
+        }
+      ]
+    }
+  ]
+}
+```
+
+#### Display examples
```

Comment:
display not working on your control, remove this example + remove from parameters table